src/actions/search.js

```diff
@@ -11,9 +11,10 @@ import { updateWallet } from './wallets';
 const searchDelegate = ({ publicKey, address }) =>
   async (dispatch, getState) => {
     const liskAPIClient = getState().peers.liskAPIClient;
+    const networkConfig = getState().network;
     const delegates = await getDelegate(liskAPIClient, { publicKey });
     const transactions = await getTransactions({
```

Comment:
You need to pass token.active to the getTransactions method

Reply:
same here